lib/cn/vblock_builder_internal.h

```diff
@@ -70,16 +80,16 @@ struct vblock_builder {
     struct cn_merge_stats *    mstats;
     struct blk_list            vblk_list;
     enum hse_mclass_policy_age agegroup;
-    u64                        vsize;
-    u64                        blkid;
-    uint                       max_size;
-    uint                       vblk_off;
+    uint64_t                   vsize;
+    uint64_t                   blkid;
+    uint32_t                   max_size;
+    unsigned int               vblk_off;
     void *                     wbuf;
-    uint                       wbuf_off;
-    uint                       wbuf_len;
-    u64                        vgroup;
+    unsigned int               wbuf_off;
+    unsigned int               wbuf_len;
+    uint64_t                   vgroup;
     bool                       destruct;
-    u32                        opt_wrsz;
+    uint64_t                   opt_wrsz;
```

Comment:
We probably should be using size_t for sizes of things.  I know we don't, but we probably should....

Reply:
And probably off_t for offsets...

Reply:
I changed the offsets to off_t. I think we should talk about size_t more broadly since currently we don't expose any size_t parameters. Nabeel and I have talked about this in the past. Happy to change mblocksz to size_t.